src/main/java/com/znsio/rpi/utils/commandline/CommandLineExecutor.java

```diff
@@ -0,0 +1,43 @@
+package com.znsio.rpi.utils.commandline;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.log4j.Logger;
+
+import java.nio.charset.StandardCharsets;
+import java.util.concurrent.TimeUnit;
+
+public class CommandLineExecutor {
+    private static final Logger LOGGER = Logger.getLogger(CommandLineExecutor.class.getSimpleName());
+    private static final int DEFAULT_COMMAND_TIMEOUT = 60;
+
+    public static CommandLineResponse execCommand(final String[] command) {
+        return execCommand(command, DEFAULT_COMMAND_TIMEOUT);
+    }
+
+    public static CommandLineResponse execCommand(final String[] command, int timeoutInSeconds) {
```

Comment:
should these be package-protected methods?

Reply:
Exposing this utlitly for consumer to execute any CLI commands and get the response accordingly.